apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -60,9 +56,17 @@ loop(Socket, Transport, State) ->
                     {ok, Ms} = metric:list(Bucket, Prefix),
                     Transport:send(Socket, dproto_tcp:encode_metrics(Ms)),
                     loop(Socket, Transport, State);
+                {info, Bucket} ->
+                    {ok, {Res, PPF, TTL}} = metric:bucket_info(Bucket),
+                    InfoBin = dproto_tcp:encode_bucket_info(Res, PPF, TTL),
+                    Transport:send(Socket, InfoBin),
```

Comment:
Remove the dynamic function call on line 62. Only modules that define callbacks should make dynamic calls.